source/2019-05-04-the-ember-times-issue-97.html

```diff
@@ -13,6 +13,7 @@
 This week:
 ...
 interview with **Yehuda Katz** 🎙️,
+the EmberFest 2019 CFP is now open 📝,
```

Comment:
I think the CFP has more importance than the interview with Yehuda (which happened a month ago). Do you want to place your announcement on top?

Reply:
Ya lets do it